node/src/chain/acceptor.rs

```diff
@@ -592,6 +592,13 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
             ffr
         );
 
+        if attested {
+            let header = tip.inner().header();
+            // Waits until next slot, if the block is attested
+            // A non-attested block suggests another winner may be on its way
+            Self::wait_until_next_slot(header.timestamp).await;
```

Comment:
Shouldn't we rely on fallback in this case?

Reply:
The chain component is sort of single-threaded reactor, blocking here due to `tokio::sleep` will not allow fallback to be executed until next slot is reached.

Reply:
Indeed, I was not suggesting to wait at all here.
And if a block is on its way it will be handled later by the current implementation of the fallback

Reply:
I agree with @herr-seppia. We should not wait here but before first broadcasting the block, as mentioned.

Reply:
For the record
I'm ok with this approach too, but in this case we should ALWAYS wait (no matter if attested or not)

Ps: more I look at this approach and more I prefer it over my suggestion

Reply:
Refactoring the PR to introduce the delay in Block generator is doable and easy.

@fed-franz @herr-seppia We'll pair up to finalize this. 

Not sure how suitable would be also to (at least file an issue) to consider VDF at some point in future.

Reply:
Agreed on waiting before validation